internal/api/router.go

```diff
@@ -1,12 +1,31 @@
 package api
 
 import (
+	"context"
+	"net/http"
+
 	"github.com/gorilla/mux"
+	get "github.com/joshsoftware/profile_builder_backend_go/internal/api/GET"
+	post "github.com/joshsoftware/profile_builder_backend_go/internal/api/POST"
 	"github.com/joshsoftware/profile_builder_backend_go/internal/app"
 )
 
-func NewRouter(deps app.Dependencies) *mux.Router{
-	router := mux.NewRouter();
-	
+// NewRouter returns a object that contains all routes of application
+func NewRouter(ctx context.Context, deps app.Dependencies) *mux.Router {
+	router := mux.NewRouter()
+
+	//POST APIs
+	router.HandleFunc("/profiles", post.CreateProfileHandler(ctx, deps.ProfileService)).Methods(http.MethodPost)
```

Comment:
use this endpoint `router.HandleFunc("/", post.CreateProfileHandler(ctx, deps.ProfileService)).Methods(http.MethodPost)`

after `subrouter := router.PathPrefix("/profiles").Subrouter()`